backend/src/routes.js

```diff
@@ -1,5 +1,11 @@
 const express = require('express');
 
+const UserController = require('./controllers/UserController');
+
 const routes = express.Router();
 
-module.exports = routes;
\ No newline at end of file
+routes.post('/createUser', UserController.create);
+routes.delete('/deleteUser/:id', UserController.delete);
```

Comment:
Question: in a security point of view, is there a difference to pass the id as a url's parameter, as a header's field or in the request's body?

Reply:
This kind a convention, that the HTTP Delete doesn't have a body.